docker-compose.yaml

```
@@ -18,36 +66,64 @@ services:
       start_period: 30s
 
 
-  redis:
-    image: "redis:8-alpine"
-    ports: 
-      - 6379:6379
+  rabbitmq:
+    image: rabbitmq:3.12-management
+    ports:
+      - "5672:5672"      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)